deepchecks/base/check.py

```diff
@@ -9,30 +9,36 @@
 # ----------------------------------------------------------------------------
 #
 """Module containing all the base classes for checks."""
-# pylint: disable=broad-except
+# pylint: disable=broad-except,inconsistent-quotes
```

Comment:
why disable this lint?

Reply:
I wanted to disable it for two specific lines, but when I added directive at the end of the lines it caused `line too long` error.
Yup, you are right) it is better to use it only for that lines, fixed